menu/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Menu DOM</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200&icon_names=menu" />
+</head>
+<body>
+    <header>
+        <div class="topnav">
+            <button onclick="openMenu()">
+                <span class="material-symbols-outlined">menu</span>
+            </button>    
+        </div>
+    </header>
+
+    <div id="toggleMenu" style="display: none;">
+        <ul>
+            <li><a href="#Letters" onclick="switchMenu('letters'); event.preventDefault();">Letters</a></li>
+            <li><a href="#Numbers" onclick="switchMenu('numbers'); event.preventDefault();">Numbers</a></li>
```

Comment:
When you add an onclick, anything can become clickable, so you don't have to use an `a` tag. And the `preventDefault` method. 

```suggestion
            <li onclick="switchMenu('letters')">Letters</li>
            <li onclick="switchMenu('numbers')">Numbers</li>
```

EDIT: I recognize the instructions said to use two links, sorry.